src/app/components/core/settings/settings.component.ts

```diff
@@ -21,6 +21,8 @@ export class SettingsComponent implements OnInit {
   refreshInterval: string;
   serviceUrl = '';
   selectedTheme: string;
+  version;
+  scheme;
```

Comment:
There is no need of another variable `scheme` as we already have global variable `protocol` for scheme.

Reply:
We have `protocol` variable in the way that it's value changed with the change in the protocol dropdrown, even without clicking save. So we can't use `protocol` variable here. cc/ @praveen-garg